cmd/vc-rest/startcmd/start.go

```diff
@@ -395,13 +402,17 @@ func startEdgeService(parameters *vcRestParameters, srv server) error {
 		externalHostURL = parameters.hostURLExternal
 	}
 
+	crypto, err := tinkcrypto.New()
+	if err != nil {
+		return err
+	}
+
 	vcService, err := vc.New(&operation.Config{StoreProvider: edgeServiceProvs.provider,
 		KMSSecretsProvider: edgeServiceProvs.kmsSecretsProvider,
-		EDVClient:          edv.New(parameters.edvURL, edv.WithTLSConfig(tlsConfig)),
-		LegacyKMS:          legacyKMS,
-		KeyManager:         localKMS,
-		VDRI:               vdri, HostURL: externalHostURL, Mode: parameters.mode, Domain: parameters.blocDomain,
-		TLSConfig: tlsConfig})
+		EDVClient:          edv.New(parameters.edvURL, edv.WithTLSConfig(&tls.Config{RootCAs: rootCAs})),
+		LegacyKMS:          legacyKMS, KeyManager: localKMS,
```

Comment:
why still legacyKMS

Reply:
@DRK3 @Baha-sk add your comments for JWE

Reply:
JWE still uses LegacyKMS @troyronda